src/patternfly/components/ChipGroup/chip-group.scss

```diff
@@ -60,5 +78,37 @@
   > .pf-c-chip {
     margin-right: var(--pf-c-chip-group--c-chip--MarginRight);
     margin-bottom: var(--pf-c-chip-group--c-chip--MarginBottom);
+
+    &:last-child {
+      --pf-c-chip-group--c-chip--MarginRight: 0;
+    }
+  }
+}
+
+.pf-c-chip-group__action {
```

Comment:
should we add an "xs" `margin-right` to this to retain the extra space on the right side of the chip-group?

To illustrate, the extra space on the right here

<img width="117" alt="Screen Shot 2019-10-23 at 12 53 57 PM" src="https://user-images.githubusercontent.com/35148959/67420426-38e4e280-f594-11e9-8cb2-65a5a2ab7114.png">

isn't here

<img width="164" alt="Screen Shot 2019-10-23 at 12 53 53 PM" src="https://user-images.githubusercontent.com/35148959/67420444-3edac380-f594-11e9-92c9-e57b8fa8e9f4.png">